variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "vpn_gateway_id" {
+  description = "The id of the VPN Gateway"
+}
+
+variable "customer_gateway_id" {
+  description = "The id of the Customer Gateway"
+}
+
+variable "create_vpn_connection" {
+  description = "Set to false to prevent the creation of a VPN Connection"
+  default     = true
+}
+
+variable "vpc_id" {
+  description = "The id of the VPC where the VPN Gateway lives"
+}
+
+variable "vpc_subnet_route_table_ids" {
+  description = "The ids of the VPC subnets for which routes from the VPN Gateway will be propagated"
+  type        = "list"
+  default     = []
+}
+
+variable "tags" {
+  description = "List fo tags to be added to the VPN Connection resource (only if `create_vpn_connection = true`)"
+  type        = "list"
```

Comment:
`tags` should be type of `map`. [Read here](https://www.terraform.io/docs/configuration/interpolation.html#merge-map1-map2-)